CONTRIBUTING.md

```diff
@@ -23,3 +23,34 @@ use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
+
+## Building Dagger
+
+Dagger is built with [`bazel`](https://bazel.build).
+
+### Building Dagger from the command line
+
+* Go to the [Bazel installation page](https://docs.bazel.build/versions/master/install.html) and install Bazel.
+  * Learn more about Bazel targets [here](https://docs.bazel.build/versions/master/build-ref.html).
+* Clone the Dagger repository: `git clone git@github.com:google/dagger.git dagger`
+* Build the Dagger project with `bazel build`
+  * If you get the following error: `ERROR: missing input file '@androidsdk//:build-tools/26.0.2/aapt'`, install the missing build tools version with sdkmanager.
```

Comment:
Full example error for missing SDK:

```
❯ bazel test //...
INFO: Invocation ID: a1ecc54e-f201-4168-8292-a562f2847e57
INFO: SHA256 (https://github.com/bazelbuild/bazel-skylib/archive/9430df29e4c648b95bf39a57e4336b44a0a0582a.zip) = a260db2d0c73f863c8b0952a8d33c3586cb4bf2902ea99c6a32be59420dc2aa0
ERROR: /Users/zsweers/dev/java/contributing/dagger/java/dagger/android/BUILD:84:1: every rule of type javadoc_library implicitly depends upon the target '@androidsdk//:platforms/android-26/android.jar', but this target could not be found because of: no such package '@androidsdk//': Android SDK api level 26 was requested but it is not installed in the Android SDK at /Users/zsweers/dev/android/android-sdk. The api levels found were [28, 27]. Please choose an available api level or install api level 26 from the Android SDK Manager.
ERROR: Analysis of target '//java/dagger/android:android-javadoc' failed; build aborted: no such package '@androidsdk//': Android SDK api level 26 was requested but it is not installed in the Android SDK at /Users/zsweers/dev/android/android-sdk. The api levels found were [28, 27]. Please choose an available api level or install api level 26 from the Android SDK Manager.
INFO: Elapsed time: 1.742s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (35 packages loaded, 168 targets configured)
FAILED: Build did NOT complete successfully (35 packages loaded, 168 targets configured)
    Fetching @androidsdk; fetching
    Fetching @local_config_cc; Restarting.
    Fetching @local_jdk; fetching
```

Full example error for missing build tools:

```
❯ bazel test //... --incompatible_package_name_is_a_function=false
INFO: Invocation ID: 396e270a-9796-40c5-bf3b-82ffacf214c2
INFO: Analysed 497 targets (191 packages loaded, 3029 targets configured).
INFO: Found 111 targets and 386 test targets...
ERROR: missing input file '@androidsdk//:build-tools/26.0.2/aapt'
ERROR: /private/var/tmp/_bazel_zsweers/1825c3febee87f83f0190e5ba08c9307/external/androidsdk/BUILD.bazel:8:1: @androidsdk//:aapt_binary: missing input file '@androidsdk//:build-tools/26.0.2/aapt'
ERROR: /private/var/tmp/_bazel_zsweers/1825c3febee87f83f0190e5ba08c9307/external/androidsdk/BUILD.bazel:8:1 1 input file(s) do not exist
INFO: Elapsed time: 17.661s, Critical Path: 2.57s
INFO: 60 processes: 60 darwin-sandbox.
FAILED: Build did NOT complete successfully
//javatests/dagger:internal/DoubleCheckTest                           NO STATUS
... (all the rest of the targets)
```